integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -35,347 +37,433 @@ setup() {
 	done
 	set -e
 	docker rm -f aerospike
+	docker rmi $image2
```

Comment:
maybe we can add an:

```bash
[ -n "$image2" ] docker rmi "$image2"
```

to the end where we do the other docker rm. In fact, maybe we should predicate the `$image` rm with a `-n` as well, just in case ;-)